### PR TITLE
Update webrtc_to_hls to be similiar with membrane_videoroom

### PR DESCRIPTION
**Pull request by Rados13**
This PR will wait for the release of rtc_engine as it relies on changes made in https://github.com/membraneframework/membrane_rtc_engine/pull/137

### CODE REVIEW

File: webrtc_to_hls/config/runtime.exs [hunk 1]
```diff
@@ -1,65 +1,134 @@
...
-      port: System.fetch_env!("PORT") |> String.to_integer()
-    ]
+config :membrane_webrtc_to_hls_demo,
+  integrated_turn_ip:
+    System.get_env("INTEGRATED_TURN_IP", "127.0.0.1") |> ConfigParser.parse_integrated_turn_ip(),
+  integrated_turn_port_range:
+    System.get_env("INTEGRATED_TURN_PORT_RANGE", "50000-59999")
+    |> ConfigParser.parse_integrated_turn_port_range(),
+  integrated_tcp_turn_port:
+    System.get_env("INTEGRATED_TCP_TURN_PORT")
```
Comment (daniel-jodlos) [inline webrtc_to_hls/config/runtime.exs:52]: If there is no default, maybe 
```suggestion
    System.fetch_env!("INTEGRATED_TCP_TURN_PORT")
```

File: webrtc_to_hls/lib/webrtc_to_hls/stream.ex [hunk 2]
```diff
@@ -131,8 +198,30 @@ defmodule WebRTCToHLS.Stream do
...
+    else
+      Engine.remove_peer(state.rtc_engine, state.peer.id)
+      state = %{state | peer: nil, channel_pid: nil}
+      {:noreply, state}
+    end
+  end
+
+  defp create_context(name) do
+    metadata = [
+      {:"library.language", :erlang},
```
Comment (daniel-jodlos) [inline webrtc_to_hls/lib/webrtc_to_hls/stream.ex:213]: erlang?

File: webrtc_to_hls/lib/webrtc_to_hls/application.ex [hunk 3]
```diff
@@ -2,18 +2,70 @@ defmodule WebRTCToHLS.Application do
...
   def start(_type, _args) do
+    config_common_dtls_key_cert()
+    create_integrated_turn_cert_file()
+
     children = [
+      {Reporter, [metrics: Membrane.RTC.Engine.Metrics.metrics(), name: WebRTCToHLSReporter]},
       WebRTCToHLSWeb.Endpoint,
       {Phoenix.PubSub, name: WebRTCToHLS.PubSub},
-      {Registry, keys: :duplicate, name: WebRTCToHLS.Registry}
+      {Registry, keys: :duplicate, name: WebRTCToHLS.Room.Registry}
```
Comment (daniel-jodlos) [inline webrtc_to_hls/lib/webrtc_to_hls/application.ex:21]: Do we even use it anywhere?

File: webrtc_to_hls/assets/tailwind.config.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    },
+  },
+  variants: {
+    extend: {
+      opacity: ["disabled"],
+    },
+    backgroundColor: ({ after }) => after(["disabled", "group-disabled"]),
+  },
+  plugins: [require("@tailwindcss/aspect-ratio")],
+};
```
Comment (daniel-jodlos) [inline webrtc_to_hls/assets/tailwind.config.js:40]: Are we using tailwind anywhere?

**Review by daniel-jodlos — CHANGES_REQUESTED** [review]

File: webrtc_to_hls/config/dev.exs [hunk 5]
```diff
@@ -1,25 +1,32 @@
...
-    dirs: [
-      "priv/static",
-      "lib/videoroom_web/controllers",
-      "lib/videoroom_web/views",
-      "lib/videoroom_web/templates"
+    patterns: [
+      ~r"priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$",
+      ~r"priv/gettext/.*(po)$",
+      ~r"lib/WebRTCToHLS_web/(live|views)/.*(ex)$",
+      ~r"lib/WebRTCToHLS_web/templates/.*(eex)$"
```
[the remark this answers is not in the document]
Comment (Rados13) [reply webrtc_to_hls/config/dev.exs:28]: Removing it is causing some errors.

**Pull request merged**